WebJobs.Script.sln

```diff
@@ -430,6 +430,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTriggerShared", "HttpTr
 		sample\HttpTriggerShared\function.json = sample\HttpTriggerShared\function.json
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ServiceBusListenerException", "ServiceBusListenerException", "{1AC76291-23CF-4698-8AC7-F8C27FCC9460}"
+	ProjectSection(SolutionItems) = preProject
+		sample\ServiceBusListenerException\function.json = sample\ServiceBusListenerException\function.json
```

Comment:
For test scenario functions like this, we put them in the e2e tests, not as samples. We try to keep the samples as actual working examples.